spec/controllers/shares_controller_spec.rb

```diff
@@ -150,6 +150,16 @@
     delete :destroy, id: share.id
     expect(response).to redirect_to shares_path
    end
+   
+   it "should allow an admin to destroy shares" do
+    admin = FactoryGirl.create(:user)
+    admin.admin = true
+    admin.save!
```

Comment:
As a heads up:

```
admin = FactoryGirl.create(:user, admin: true)
```

Does the same as the previous 3 lines in a single one.  Can be useful to shorten things.